docs/source/operations/quorum_networkyaml.md

```diff
@@ -132,6 +142,7 @@ The fields under `config` are
 | tm_nodes | The Transaction Manager nodes public addresses should be provided. For `tessera`, all participating nodes should be provided, for `constellation`, only one bootnode should be provided. NOTE The difference in the addresses for Tessera and Constellation. |
 | staticnodes | *** Existing network's static nodes need to be provided as an array. This will be implemented with addition of new node. |
 | genesis | *** Existing network's genesis.json needs to be provided in base64. This will be implemented with addition of new node.|
+| geth | *** Geth account details . This will be implemented with addition of new node.|
```

Comment:
"This will be implemented with .." This text should not be replaced with actual implementation. You need to provide details of each parameter under geth. Also I think the name "geth" is a bit confusing for this parameter, change it to "bootnode"